.yamllint

```diff
@@ -0,0 +1,14 @@
+---
+ignore: |
+  /lib/generators/**/templates/**
```

Comment:
Minor, but you don't have generators here, not spec/tools/openstack_data, so those lines can be removed.